lib/internal/Magento/Framework/App/Bootstrap.php

```diff
@@ -279,7 +279,6 @@ protected function assertMaintenance()
         if (null === $isExpected) {
             return;
         }
-        $this->initObjectManager();
         /** @var \Magento\Framework\App\MaintenanceMode $maintenance */
         $this->maintenance = $this->objectManager->get(\Magento\Framework\App\MaintenanceMode::class);
```

Comment:
@Bartlomiejsz  i think we can remove  Maintenance initialization  from constructor .
I don't know why Maintenance initialization happens in two times (assertMaintenance method,initObjectManager method)
i think Its enough in assertMaintenance.
anyway I'm looking for core team feedback?